### ISSUE TITLE
Move the compile-time configuration information into the library

### ISSUE THREAD

**Issue by bcardarella** [liveview-native/kino_live_view_native#1]
I'd like to find a way to move the compile-time configuration from the `Mix.install` in the Livebook to within this library itself. I had originally gone with `config/config.exs` but of course this won't load when this lib is pulled in as a dependency. There is also setting the `env` within the `application` function for `Mix.Project`:  https://hexdocs.pm/mix/Mix.Tasks.Compile.App.html#content and https://hexdocs.pm/elixir/Application.html#module-the-application-environment but this will only set default values for the application in question, I also need to configure `live_view_native` platform.

The config files cannot be read in at runtime from what I can tell. Even if this isn't performant I'd be OK with this if there was a work around because the intent of this library is for Livebook tutorials and not production use.

**Comment by NduatiK** [comment]
I think we can get around by only starting the phoenix server when we are ready to serve requests.

We can:
1. Remove server startup code from `KinoLiveViewNative.Application`
2. Provide a function `KinoLiveViewNative.start_server(applicationName, port)`. 
When called, the function sets necessary configs using `Application.put_env` then starts Phoenix.
 
Users can then call `KinoLiveViewNative.start_server(...)` at the top of the notebook to get all the configs.

WDYT?

**Comment by NduatiK** [comment]
Got this working. Users just add this to the top of their notebook.
```elixir
KinoLiveViewNative.start(app_name: "LiveView Native", port: 4000)
```

I can create a PR once #2 is merged in.

**Comment by NduatiK** [comment]
@bcardarella, let me know if I can close this. The issue should have been addressed with #2

**Comment by bcardarella** [comment]
:+1:
